src/Compression/ICompressionCodec.h

```diff
@@ -112,6 +112,9 @@ class ICompressionCodec : private boost::noncopyable
     /// If it does nothing.
     virtual bool isNone() const { return false; }
 
+    /// This is a knob for Deflate QPL codec.
```

Comment:
Please move this below `isExperimental()` (l. 110).

Reply:
Done